index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+     
+         <div class="header">
+               <div class="logo">
+                   <h1>My Logo</h1>
+               </div>
+               <div class="navbar-item">
+                   <a href="#">Portfolio</a>
+                   <a href="#">About</a>
+                   <a href="#">contact</a>
+                   <a href=""><img src="/images/Rec.png" alt=""></a>
+                   <img src="/images/Union.png" alt="">
+                   <img class="icon" src="/images/Union1.png" alt="">
+               </div>
+         </div>
+
+   <section>
+    <div class="about">
+        <div class="social">
+            <img src="/images/github (1).svg" alt="">
+            <img src="/images/Linkedin icon.svg" alt="">
+            <img src="/images/angel list.svg" alt="">
+            <img src="/images/twitter.svg" alt="">
+            <img src="/images/medium.svg" alt="">
+        </div>
```

Comment:
- [ ] Please I suggest wrapping the social icons with `ul` tag and 'li' tag for each icon. Also please add links with `a` tags to each icon. 😄 
Example: 

```
<div class="social">
            <ul>
            <li><a href="#"><img src="/images/github (1).svg" alt=""></a></li>
            <li><a href="#"><img src="/images/Linkedin icon.svg" alt=""></a></li>
            <li><a href="#"><img src="/images/angel list.svg" alt=""></a></li>
            <li><a href="#"><img src="/images/twitter.svg" alt=""></a></li>
            <li><a href="#"><img src="/images/medium.svg" alt=""></a></li>
            </ul>
            
        </div>
```